tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/prep-abort-test.go

```diff
@@ -32,10 +31,16 @@ var _ = Describe(
 
 		AfterEach(func() {
 			By("Deleting IBGU on target hub cluster", func() {
-				_, err := newIbguBuilder.DeleteAndWait(1 * time.Minute)
-				Expect(err).ToNot(HaveOccurred(), "Failed to delete IBGU on target hub cluster")
-			})
+				newIbguBuilder := ibgu.NewIbguBuilder(cnfinittools.TargetHubAPIClient,
```

Comment:
You can remove the extra `With` functions if you're just going to delete right after creating the builder since all it needs is the name and namespace to delete. You could also combine it if you want so you get
```go
_, err := ibgu.NewIbguBuilder(cnfinittools.TargetHubAPIClient, tsparams.IbguName, tsparams.IbguNamespace).DeleteAndWait(time.Minute)
```
but either way works

Reply:
Thanks, Kirsten. This makes sense. I updated the PR.